### PR TITLE
Presets fix: removed old event listener that caused console errors

### PR DESCRIPTION
**Pull request by limonspb**
Removed a piece of code from the medieval ages that was causing console errors.

The error pointed by @haslinghuis [here](https://github.com/betaflight/betaflight-configurator/pull/2618?notification_referrer_id=NT_kwDOACyh47IyNDk1ODkyMDE5OjI5MjUwMjc#issuecomment-1367072553)

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by sugaarK — APPROVED** [review]

**Comment by haslinghuis** [comment]
Watching workflow - waiting for nightly to finish

**Pull request merged**
